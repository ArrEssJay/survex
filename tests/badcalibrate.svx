*fix 1 reference 0 0 0
; A scale factor of 0 isn't sensible
*calibrate compass 1.0 0
*calibrate clino backclino -0.5 0
*calibrate tape 0 0
