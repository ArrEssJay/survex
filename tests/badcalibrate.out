./badcalibrate.svx:4:25: error: Scale factor must be non-zero
./badcalibrate.svx:5:34: error: Scale factor must be non-zero
./badcalibrate.svx:6:20: error: Scale factor must be non-zero
./badcalibrate.svx:8: error: Can't calibrate angular and length quantities together

Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...

Calculating statistics...

Survey contains 1 survey station, joined by 0 legs.
There are 0 loops.
Total length of survey legs =    0.00m (   0.00m adjusted)
Total plan length of survey legs =    0.00m
Total vertical length of survey legs =    0.00m
   1 0-node.
There were 0 warning(s) and 4 error(s) - no output files produced.
